### ISSUE TITLE
brush size does not seem to work on IOS

### ISSUE THREAD

**Issue by samccone** [samccone/moji-brush#17]
it is extra small for some reason...

**Comment by joryphillips** [comment]
we should try adding "Arial, ArialMT" and see if that fixes it

**Comment by joryphillips** [comment]
Turns out it's not a matter of Arial not being on iOS like I thought, but rather it's an issue of iOS Webkit not allowing emoji bigger than 16px. [There's a fix in the works](https://bugs.webkit.org/show_bug.cgi?id=148265).

**Comment by samccone** [comment]
ugh terrrrrrrible, what to do... what to do....

**Comment by joryphillips** [comment]
I suppose an option is switching to embedding them as images. Maybe base64. Obviously makes for a bigger download but we could limit the palette, and I don't think it would be too hard to switch to that method.

**Comment by samccone** [comment]
i would like it if this was the fallback and only downloaded if we were in this broken state. :(

**Comment by samccone** [comment]
@paulirish says we should try transforming it up

right now we are transforming it down... I think if we invert it we should be solid

**Comment by samccone** [comment]
yeah so we can transform / zoom it up but painting is the hard part...

unfortunately I think that we are going to have to have a fallback emoji map for IOS :(
